submissions/kolawoleOgunfowokan/PHPTask/php/overview_helper.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
In Overview, you are fetching directly from the DB so you do not need this file. You can remove it